florist/tests/integration/api/launchers/test_launch.py

```diff
@@ -8,9 +8,9 @@
 import torch
 from fl4health.server.base_server import FlServer
 
-from florist.api.launchers.launch import launch
-from florist.tests.utils.api.fl4health_utils import MnistClient, get_server_fedavg
-from florist.tests.utils.api.models import MnistNet
+from florist.api.launchers.local import launch
```

Comment:
good call, I like the distinction between local and "distributed" launchers that we will build out down the line